src/resources/postcss/views/full/month/_mobile-events.pcss

```diff
@@ -7,7 +7,7 @@
 	.tribe-events-calendar-month-mobile-events__mobile-event {
 		border-bottom: 1px solid var(--color-border-default);
 
-		&:last-child {
+		&:last-of-type {
```

Comment:
`:last-of-type` is used for elements, not classes. what are you trying to do here?

Reply:
I'm trying to remove the border of the last element of that type. If you check the screencast you'll see that the last element doesn't have border and then the `+X read more` comes.

Reply:
gotcha. so maybe what we want to do is rather than have a border bottom, we can have a border top on `.tribe-events-calendar-month-mobile-events__mobile-event + .tribe-events-calendar-month-mobile-events__mobile-event`.